### DISCUSSION TITLE
Move my pool from one machine to another (same drives)

### DISCUSSION THREAD

**Discussion by psyciknz** [trapexit/mergerfs#1243] [Q&A] [answered]
So I have a mergerFS system running on an openmediavault VM where I pass the drives from a host.

I'd like to drop the open media vault part and run mergerfs directly on the host (find the omv server can get a bit overloaded).

Is it just a matter of replicating the fstab entries on the host and getting it to just go, or would there be some other tasks I need to follow to get the host to pick up the pool.

**Comment by trapexit** [comment, marked as answer]
Just run mergerfs as shown in the docs. The main point of mergerfs is that it doesn't do anything special with the underlying filesystem. Just mount the filesystems on the host and setup a pool.

**Reply by psyciknz** [reply]
thanks.  Was one of the features I was most interested in.
